packages/agents/relayer/src/bindings/relays/index.ts

```diff
@@ -89,16 +89,41 @@ export const pollCache = async () => {
         continue;
       }
 
-      const { data, to } = task;
+      const { data, to, keeper } = task;
 
       // TODO: Queue up fee claiming for this transfer after this (assuming transaction is successful)!
       try {
-        const transaction = {
+        // Define transaction
+        let transaction = {
           domain,
           to,
           data,
           from: await wallet.getAddress(),
         };
+
+        if (keeper) {
+          // NOTE: vault holds klp liquidity, and is the job address. have to bond this to job via the keep3r UI
+          // verify keeper support is valid on this chain
+
+          // lookup relay address (consistent across chains, minus zksync)
+          // lookup automation vault
+          const { automationVault, xKeeperRelayer } = config.chains[domain].deployments;
+          if (!automationVault || !xKeeperRelayer) {
+            throw new Error(`No automationVault or xkeeperRelayer for domain ${domain}`);
+          }
+
+          // call keeperRelay.exec with the transaction data
+          // TODO: batch exec calls from cache that are keeper flagged
+          // TODO: fallthrough if keeper sending fails. likely in cache
```

Comment:
We could handle fallback at the caller level `sendWithRelayerWithBackup` so that it tries in this order:
1 Connext Relayer using Keep3r network, setup Keep3r api endpoint/flag on relayer HTTP server
2 Gelato
3 Connext Relayer as hot wallet.

Reply:
that's the idea

we can prioritize keeper for prove and process, which is where we will see the most gains